src/ui/webeiddialog.cpp

```diff
@@ -553,7 +553,7 @@ void WebEidDialog::resizeHeight()
     adjustSize();
 }
 
-QPixmap WebEidDialog::pixmap(const QLatin1String& name) const
+QPixmap WebEidDialog::pixmap(QLatin1String name) const
```

Comment:
C++ Core Guideline F.16: [_For “in” parameters, pass cheaply-copied types by value and others by reference to const_](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#f16-for-in-parameters-pass-cheaply-copied-types-by-value-and-others-by-reference-to-const). I guess you want to rely on the move constructor here, but it is hard to see the justification. Do you want to justify this?

Reply:
https://github.com/KDE/clazy/blob/1.11/docs/checks/README-function-args-by-value.md

Reply:
OK, I found the explanation from [Thiago Macieira's blog](https://www.macieira.org/blog/2012/02/the-value-of-passing-by-value/):

_In the [String Theory](http://labs.qt.nokia.com/2008/04/28/string-theory/) blog, I concluded that a small structure like QLatin1String, which contained exactly one pointer as a member, would benefit from passing by value._

Fair enough then I guess.